src/modules/endpoint/ResourceURLFinder.js

```diff
@@ -5,9 +5,14 @@ import { Parser } from 'htmlparser2'
 import type { FetchMapType } from './sagas/fetchResourceCache'
 import { reduce } from 'lodash'
 import hashUrl from './hashUrl'
-import { ExtendedPageModel } from '@integreat-app/integreat-api-client'
 import Url from 'url-parse'
 
+interface InputEntryType {
```

Comment:
Why use `interface` here? What is the difference?

Reply:
https://flow.org/en/docs/types/interfaces/
The benefit is, that I can test the method by only passing objects that have the necessary properties and so I don't have to create ExtendedPageModels with all the attributes that are not needed for (testing) this method.

Reply:
Should we mention this in the model builder issue then to change it?

Reply:
Not sure what you mean by that

Reply:
This (the use of interface) will be obsolete once we have model builders, right? If so, we could add a note to the model builder issue you created.

Reply:
Hmmm, no I wouldn't adjust this. Imo this is equally good or even better than using modelBuilders and ExtendedPageModel here, as we only need the 3 attributes defined in the interface for using the method...